### PR TITLE
EPMDJ-8111 Support kafka for autotest agent

### PR DESCRIPTION
**Pull request by LikhanovMaxim**
changed instrumentation of consumer for Spring

### CODE REVIEW

File: src/commonMain/kotlin/com/epam/drill/Constants.kt [hunk 1]
```diff
@@ -21,3 +21,6 @@ const val SYSTEM_CONFIG_PATH = "DRILL_AGENT_CONFIG_PATH"
 const val SYSTEM_HTTP_HOOK_ENABLED = "DRILL_HTTP_HOOK_ENABLED"
 const val SYSTEM_JAVA_APP_JAR = "JAVA_APP_JAR"
 const val HTTP_HOOK_ENABLED = "drill.http.hook.enabled"
+
+const val KAFKA_PRODUCER_INTERFACE = "org/apache/kafka/clients/producer/Producer"
+const val KAFKA_CONSUMER_SPRING = "org/springframework/kafka/listener/KafkaMessageListenerContainer\$ListenerConsumer"
```
Comment (Mazel-Tovr) [inline src/commonMain/kotlin/com/epam/drill/Constants.kt:26]: Will this work for the consumer on [org.apache.kafka](https://mvnrepository.com/artifact/org.apache.kafka) and not only on [springframework.kafka](https://mvnrepository.com/artifact/org.springframework.kafka) ?

File: src/jvmMain/kotlin/com/epam/drill/agent/instrument/KafkaTransformer.kt [hunk 2]
```diff
@@ -35,8 +36,8 @@ actual object KafkaTransformer {
         return try {
             ClassPool.getDefault().appendClassPath(LoaderClassPath(loader as? ClassLoader))
             when (className) {
-                "org/apache/kafka/clients/producer/Producer" -> producerInstrument(classfileBuffer)
-                "org/apache/kafka/clients/consumer/Consumer" -> consumerInstrument(classfileBuffer)
+                KAFKA_PRODUCER_INTERFACE -> producerInstrument(classfileBuffer)
+                KAFKA_CONSUMER_SPRING -> consumerInstrument(classfileBuffer)
```
Comment (LikhanovMaxim) [inline src/jvmMain/kotlin/com/epam/drill/agent/instrument/KafkaTransformer.kt:40]: add todo Consumer only for Spring

File: src/commonMain/kotlin/com/epam/drill/Constants.kt (hunk 1 above)
Comment (LikhanovMaxim) [reply src/commonMain/kotlin/com/epam/drill/Constants.kt:26]: it will work only for  springframework.kafka

Comment (Mazel-Tovr) [reply src/commonMain/kotlin/com/epam/drill/Constants.kt:26]: I mean, do we need org.apache.kafka support?

Comment (LikhanovMaxim) [reply src/commonMain/kotlin/com/epam/drill/Constants.kt:26]: for current needs no. In the future, it will need to implement

**Pull request merged**